src/apps/loan-list/materials/utils/warning-bar.tsx

```diff
@@ -48,11 +52,15 @@ const WarningBar: FC<WarningBarProps> = ({
           >
             {rightText}
           </p>
+          <span className="hide-visually" id={labelId}>
+            {rightButtonAriaLabelText}
+          </span>
           {rightLink && (
             <Link
```

Comment:
Wouldn't it be better to use the LinkButton component here?

Reply:
@JacobArrow does it matter? It is only a label. And it's not visible at all anyways.